### PR TITLE
Service template option

### PR DESCRIPTION
**Pull request by moritzfriedrich**
Add is_template option in Object Service class and adjust the service template.

### CODE REVIEW

**Comment by janvonde** [comment]
+1 for this merge...!

**Comment by moritzfriedrich** [comment]
i added template options for host and user too
and fixed some issues in the each-loop for commands of several templates

**Comment by lazyfrosch** [comment]
@moritzfriedrich thanks for your efforts, we would have to rebase that branch against current develop, I moved the templates to a sub dir, and introduced a new $templates var for imports

Can you do it, or should I? :wink:

**Comment by moritzfriedrich** [comment]
@lazyfrosch I rebased and merged the template option into the current develop branch

**Comment by arioch** [comment]
@moritzfriedrich please also flatten (rebase fixup) your changes to a single commit.

File: templates/object/checkcommand.conf.erb [hunk 1]
```diff
@@ -15,7 +15,7 @@ object CheckCommand "<%= @object_checkcommandname %>" {
   import "<%= t -%>"
   <%- end -%>
   <%- if @command -%>
-  command = [ <% if @cmd_path -%><%= @cmd_path -%> + <% end -%><% @command.each_with_index do |cmd, i| %>"<%= cmd -%>"<%= ', ' if i < (@command.size - 1) %><% end %> ]
+  command = [ <% @command.each_with_index do |cmd, i| %><% if @cmd_path -%><%= @cmd_path -%> + <% end -%>"<%= cmd -%>"<%= ', ' if i < (@command.size - 1) %><% end %> ]
```
Comment (lazyfrosch) [inline templates/object/checkcommand.conf.erb:18]: Is this an intended change here?

File: templates/object/notificationcommand.conf.erb [hunk 2]
```diff
@@ -16,7 +16,7 @@ object NotificationCommand "<%= @object_notificationcommandname %>" {
   <%- end -%>
   <%- if @command -%>
 
-  command = [ <% if @cmd_path -%><%= @cmd_path -%> + <% end -%><% @command.each_with_index do |cmd, i| %><%= cmd -%><%= ', ' if i < (@command.size - 1) %><% end %> ]
+  command = [ <% @command.each_with_index do |cmd, i| %><% if @cmd_path -%><%= @cmd_path -%> + <% end -%>"<%= cmd -%>"<%= ', ' if i < (@command.size - 1) %><% end %> ]
```
Comment (lazyfrosch) [inline templates/object/notificationcommand.conf.erb:19]: Is this an intended change here? (2)

**Comment by moritzfriedrich** [comment]
Please wait just a moment until i fixed my mess.

**Comment by lazyfrosch** [comment]
Ok, make it so

**Comment by moritzfriedrich** [comment]
I made a new and clean branch: https://github.com/Icinga/puppet-icinga2/pull/178

**Comment by lazyfrosch** [comment]
You could have just force pushed to the existing branch :)

**Pull request closed without merging**